roles/Maintainers.md

```diff
@@ -1,3 +1,27 @@
 # Maintainers
 
 The Maintainers roles and responsibilities are listed in [CONTRIBUTOR-ROLES.md](/CONTRIBUTOR-ROLES.md#maintainers). The following are the current maintainers:
+
+## Cilium
```

Comment:
What about cilium/proxy?

Reply:
I've added it below. Are there any other ones you think we should add?

Reply:
Yes, I thought I saw it before, cilium/cilium-cli

Reply:
I've added Tobias there. I think that is it?

Reply:
Michi is also part of it https://github.com/orgs/cilium/teams/cilium-cli-maintainers

Reply:
added

Reply:
Not easily no :(